docs/api-markdown-documenter/front-matter.js

```diff
@@ -0,0 +1,115 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const {
+	createDocumentWriter,
+    getLinkForApiItem,
+    getUnscopedPackageName,
+	renderNodeAsMarkdown,
+    transformDocNode,
+	getMarkdownRenderContextWithDefaults,
+} = require("@fluid-tools/api-markdown-documenter");
+const { ApiItemKind } = require("@microsoft/api-extractor-model");
+const os = require("os");
+
+const generatedContentNotice = "[//]: # (Do not edit this file. It is automatically generated by @fluidtools/api-markdown-documenter.)";
+
+/**
+ * Creates Hugo front-matter for the given API item.
+ * This will be appended to the top of the generated API documents.
+ *
+ * @param {ApiItem} apiItem - The root API item of the document being rendered.
+ * @param {MarkdownDocumenterConfiguration} config - See
+ * {@link @fluid-tools/api-markdown-documenter#MarkdownDocumenterConfiguration}.
+ *
+ * @returns The JSON-formatted Hugo front-matter as a `string`.
+ */
+function createHugoFrontMatter(apiItem, config, customRenderers) {
```

Comment:
I thought one of the benefits of using a unist-compatible model was that we could create AST nodes that we then serialize instead of writing strings. [mdast supports frontmatter](https://github.com/syntax-tree/mdast#frontmattercontent) - could we use that?

Reply:
Hmm. That's a fair point. There isn't a real reason why this needs to return a string instead of some arbitrary AST, so long as the appropriate renderer is provided for it.

I'll make a note of this for the next version.